src/spec/example.cr

```diff
@@ -35,11 +35,13 @@ module Spec
         ensure
           Spec.run_after_each_hooks
 
-          # We do this to give a chance for signals (like CTRL+C) to be handled,
-          # which currently are only handled when there's a fiber switch
-          # (IO stuff, sleep, etc.). Without it the user might wait more than needed
-          # after pressing CTRL+C to quit the tests.
-          Fiber.yield
+          {% unless flag?(:win32) %}
```

Comment:
I guess this is because Fiber.yield is not allowed in windows. If so, then maybe is better to add the `flag?(:win32)` check inside `Fiber.yield`

Reply:
`Fiber` itself is currently undefined in win32.